src/azure-cli-core/azure/cli/core/auth/identity.py

```diff
@@ -29,6 +29,9 @@
 AZURE_CLIENT_ID = "AZURE_CLIENT_ID"
 AZURE_CLIENT_SECRET = "AZURE_CLIENT_SECRET"
 
+WAM_PROMPT = (
+    "Please select the account you want to log in with. "
+    "For more information on login with Azure CLI, see https://go.microsoft.com/fwlink/?linkid=2271136")
```

Comment:
Since we do see customers having issues with WAM (https://github.com/Azure/azure-cli/issues/28997), should we put the instruction for disabling WAM directly in this message? @dcaro